docs/manage-metadata-using-gravitino.md

```diff
@@ -865,8 +865,14 @@ tableCatalog.purgeTable(NameIdentifier.of("metalake", "catalog", "schema", "tabl
 </TabItem>
 </Tabs>
 
-There are two ways to drop a table: `dropTable` and `purgeTable`, the difference between them is that `purgeTable` will remove data of the table, while `dropTable` only removes the metadata of the table. Some engines such as 
-Apache Hive support both, `dropTable` will only remove the metadata of a table and the data in HDFS can be reused later through the format of the external table.
+There are two ways to drop a table: `dropTable` and `purgeTable`: 
```

Comment:
Optional:

`to drop a table` -> `to remove a table`

WDYT?

Reply:
ok